lib/hammer_cli_foreman/auth.rb

```diff
@@ -10,6 +10,11 @@ class LoginCommand < HammerCLI::AbstractCommand
       option ["-p", "--password"], "PASSWORD", _("password to access the remote system")
 
       def execute
+        if !(HammerCLIForeman.foreman_api_connection.authenticator.is_a?(HammerCLIForeman::Api::SessionAuthenticatorWrapper))
+          print_message(_("Can't perform login. Make sure you have sessions enabled."))
```

Comment:
perhaps "Make sure sessions are enabled in hammer configuration file" to make it more clear